### ISSUE TITLE
Key advantages in Docs

### ISSUE THREAD

**Issue by bmaclean** [acdlite/recompose#780]
Under the "key advantages" section of the docs, is the point:

> They help prevent abuse of the setState() API, favoring props instead.

still relevant, given that modern functional components could abuse the `useState` API? Just came across this and was wondering if it was outdated.

**Comment by wuct** [comment]
Indeed, some parts of Recompose’s advantages can be replaced by React hooks.

**Comment by wuct** [comment]
Considering state management, hooks are simpler (and faster?).
